web_pwa_oca/templates/assets.xml

```diff
@@ -2,23 +2,27 @@
 <odoo>
     <template id="web_layout_pwa" name="Web layout PWA" inherit_id="web.layout">
```

Comment:
I agree with https://github.com/OCA/web/pull/1776#discussion_r557772635.

The change you should do to fix your problem is this line, and revert all other changes in the PR:

```suggestion
    <template id="web_layout_pwa" name="Web layout PWA" inherit_id="web.webclient_boostrap">
```